vignettes/why-it-works.Rmd

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Interval censoring and truncation in epidemiological delay modelling"
+author: "Samuel Brand"
```

Comment:
This is fine (though perhaps package wide we want to not have authors as it becomes unclear when more people start contributing) but I actually meant to the package DESCRIPTION.

Reply:
That makes sense!